sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### Breaking Changes
 
+- Updated `MaxPageResults` type to `int32_t` affecting:
+  - GetDeletedKeysOptions.
+  - GetPropertiesOfKeysOptions.
+  - GetPropertiesOfKeyVersionsOptions.
```

Comment:
```suggestion
  - GetDeletedKeysOptions().
  - GetPropertiesOfKeysOptions().
  - GetPropertiesOfKeyVersionsOptions().
```

Reply:
Suggestion to suggestion: backtick as `GetDeletedKeysOptions()` and so on.

Reply:
@ahsonkhan, I updated your suggestion with backticks, if you don't mind.

Reply:
I don't mind at all, but looks like it wasn't applied :p